frontend/packages/console-plugin-sdk/src/typings/dashboards.ts

```diff
@@ -126,14 +117,14 @@ namespace ExtensionProperties {
   }
 
   export interface DashboardsOverviewUtilizationItem extends DashboardsExtensionProperties {
-    /** The utilization item title */
-    title: string;
+    /** The utilization item to be replaced */
+    id: string;
```

Comment:
Nit: we should also write an extension check that ensures plugins don't re-replace dashboard utilization items.

Can be done as a follow-up.